web_app/calendar/static/pages/task_creation_form.html

```diff
@@ -23,7 +23,9 @@
         Month: <input name="month" style="width:40px;" value="{month}">
         Day: <input name="day" style="width:40px;" value="{day}">
         Hour: <input name="hour" style="width:40px;" value="{hour}"></div>
-    <p><input type="submit" value="Add"></p>
+    <input type="submit" value="Add">
+    <input class="button" type="button" onclick="window.location.replace('{redirect}')" value="Cancel" />
```

Comment:
"The replace() method rewrites the current page in the history. The current page is wiped, so when you click the “back” button, you go back to the page that now is the last visited one." - I don't think we need such kind of application behavior.